### PR TITLE
Add Tangle certification blueprint

### PR DESCRIPTION
**Pull request by JakeSCahill**
This PR also included a review of the other blueprints.

### CODE REVIEW

File: blueprints/0.1/tangle-certification/overview.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+This solution leads to the following benefits: 
+
+- Immutability
+- Trusted/decentralized verification
+- Feeless management infrastructure
+
+## Demo
+
+A demonstration is not available at the moment.
```
Comment (obany) [inline blueprints/0.1/tangle-certification/overview.md]: Maybe you could direct people to the other certification stuff https://docs.iota.org/docs/utils/0.1/official/tangle-certificate/overview for a demonstration ?

File: blueprints/0.1/tangle-certification/overview.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+However, you can test the [Tangle certificate JavaScript library](root://utils/0.1/official/tangle-certificate/overview.md) that we built, using this blueprint.
```
Comment (obany) [inline blueprints/0.1/tangle-certification/overview.md]: This probably needs to change. The new lib does only the low level parts of the certification process and would need to be wrapped in an API to make it behave like the blueprint.

File: blueprints/0.1/tangle-certification/architecture.md [hunk 3]
```diff
@@ -0,0 +1,530 @@
+# Application architecture
+
+**The Tangle certificate application uses the IOTA MAM protocol to keep track of all valid certificates in streams of transactions called MAM channels.**
```
Comment (obany) [inline blueprints/0.1/tangle-certification/architecture.md]: This only uses MAM for the DID no the certificates themselves

File: blueprints/0.1/tangle-certification/architecture.md [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+
+:::warning:Disclaimer
+Running an open source project, like any human endeavor, involves uncertainty and trade-offs. We hope the architecture described below helps you to deploy similar systems, but it may include mistakes, and can’t address every situation. If you have any questions about your project, we encourage you to do your own research, seek out experts, and discuss them with the IOTA community.
+:::
```
Comment (obany) [inline blueprints/0.1/tangle-certification/architecture.md]: There will also be an additional disclaimer about GDPR etc
Julie/Felix are working on adding that to the blueprint document, we can then copy from there.

**Comment by JakeSCahill** [comment]
Closing this PR as we are no longer releasing this blueprint

**Pull request closed without merging**